### PR TITLE
Finish building the mutation level

### LINKED ISSUE #70: Implement `excludedMutations` for building the mutation level

### PR DESCRIPTION
**Pull request by dvcopae**
Closes #70

### CODE REVIEW

File: packages/instrumenter/src/mutation-level/mutation-level.ts [hunk 1]
```diff
@@ -47,9 +48,30 @@ export interface MutationLevel {
   StringLiteral?: StringLiteral[];
   UnaryOperator?: UnaryOperator[];
   UpdateOperator?: UpdateOperator[];
-  [k: string]: unknown;
+  [k: string]: MutatorDefinition[] | string | undefined;
```
Comment (Luctia) [inline packages/instrumenter/src/mutation-level/mutation-level.ts:51]: I don't think I quite understand this syntax, what does this mean?

**Review by Luctia — APPROVED** [review]
Looks good, I think, but I might be missing something, don't have much TS experience. I have one syntax question.

File: packages/instrumenter/src/mutation-level/mutation-level.ts (hunk 1 above)
Comment (dvcopae) [reply packages/instrumenter/src/mutation-level/mutation-level.ts:51]: It's an index signature for the object. Without it, you wouldn't be able to to stuff like
`
const string = 'StringLiteral';
level[string];
`

I changed it so I would avoid some type castings which were necessary with `unknown`

**Review by Ja4pp — APPROVED** [review]
Looks good!

**Pull request merged**